xml/art-nvidia-vgpu.xml

```diff
@@ -0,0 +1,1137 @@
+     </para>
+<screen>
+&prompt.user;lspci | grep NVIDIA
+84:00.0 3D controller: NVIDIA Corporation GV100GL [Tesla V100 PCIe 16GB] (rev a1)
+</screen>
+    </step>
+    <step>
+     <para>
+      Check for the mdev supported devices and detailed information:
+     </para>
+<screen>
+&prompt.user;ls /sys/bus/pci/devices/0000:84:00.0/mdev_supported_types/
+nvidia-105  nvidia-106  nvidia-107  nvidia-108  nvidia-109  nvidia-110 [...]
+</screen>
+     <para>
+      The map of vGPU mdev devices and their type is as follows:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        nvidia-105 to nvidia-109: 1Q 2Q 4Q 8Q 16Q
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        nvidia-110 to nvidia-114: 1A 2A 4A 8A 16A
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        nvidia-115, nvidia-163, nvidia-217, nvidia-247: 1B 2B 2B4 1B4
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        nvidia-299 to nvidia-301: 4C 8C 16C
+       </para>
+      </listitem>
+     </itemizedlist>
+     <para>
+      Refer to
+      <link xlink:href="https://docs.nvidia.com/grid/11.0/grid-vgpu-user-guide/index.html#vgpu-types-tesla-v100-pcie"/>
+      for more details.
+     </para>
+    </step>
+    <step>
+     <para>
+      Inspect a vGPU device:
+     </para>
+<screen>
+&prompt.user;cd /sys/bus/pci/devices/0000:03:00.0/mdev_supported_types/
+&prompt.user;ls nvidia-105
+&prompt.user;cat nvidia-105/description 
+num_heads=2, frl_config=60, framebuffer=1024M, max_resolution=4096x2160, max_instance=16
+&prompt.user;cat nvidia-105/name 
+GRID V100-1Q
+</screen>
+    </step>
+    <step>
+     <para>
+      Generate a unique ID and create an mdev device based on it:
+     </para>
+<screen>
+      &prompt.user;uuidgen
+      4f3b6e47-0baa-4900-b0b1-284c1ecc192f
+      &prompt.sudo;echo "4f3b6e47-0baa-4900-b0b1-284c1ecc192f" > nvidia-105/create
+     </screen>
+    </step>
+    <step>
+     <para>
+      Verify the new mdev device. You can inspect the content of the
+      <filename>/sys/bus/mdev/devices</filename> directory:
+     </para>
+<screen>
+&prompt.user;cd /sys/bus/mdev/devices
+&prompt.user;ls -l
+lrwxrwxrwx 1 root root 0 Aug 30 23:03 86380ffb-8f13-4685-9c48-0e0f4e65fb87 \
+ -> ../../../devices/pci0000:80/0000:80:02.0/0000:84:00.0/86380ffb-8f13-4685-9c48-0e0f4e65fb87
+lrwxrwxrwx 1 root root 0 Aug 30 23:03 86380ffb-8f13-4685-9c48-0e0f4e65fb88 \
+ -> ../../../devices/pci0000:80/0000:80:02.0/0000:84:00.0/86380ffb-8f13-4685-9c48-0e0f4e65fb88
+lrwxrwxrwx 1 root root 0 Aug 30 23:03 86380ffb-8f13-4685-9c48-0e0f4e65fb89 \
+ -> ../../../devices/pci0000:80/0000:80:02.0/0000:84:00.0/86380ffb-8f13-4685-9c48-0e0f4e65fb89
+lrwxrwxrwx 1 root root 0 Aug 30 23:03 86380ffb-8f13-4685-9c48-0e0f4e65fb90 \
+ -> ../../../devices/pci0000:80/0000:80:02.0/0000:84:00.0/86380ffb-8f13-4685-9c48-0e0f4e65fb90
+</screen>
+     <para>
+      Or you can use the <command>mdevctl</command> command:
```

Comment:
I can not find this command on SLE15SP2 server, so chapter 3 is maybe wrong saying this is : "SUSE Linux Enterprise Server 15 SP2 or newer"